CROSSDC.md

```diff
@@ -117,6 +117,14 @@ Additional configuration properties:
 
    *groupId* - the group id to give Kafka for the consumer, default to the empty string if not specified.
 
+The following additional configuration properties should either be specified for both the producer and the consumer or in the shared Zookeeper
+central config properties file. This is because the Consumer will use a Producer for retries.
+
+   *batchSizeBytes* - the maximum batch size in bytes for the queue
+   *bufferMemoryBytes* - the amount of memory in bytes allocated by the Producer in total for buffering 
+   *lingerMs* - the amount of time that the Producer will wait to add to a batch
+   *requestTimeout* - 
```

Comment:
Do you want to add a description to `requestTimeout` ?

Reply:
I'll add a note. I thought I had but maybe that was a related note to another param. You want this to be less than the time it would take for the consumer to be booted from the group for taking too long.